src/manifest.c

```diff
@@ -688,7 +688,8 @@ struct manifest *load_manifest(int current, int version, struct file *file, stru
 {
 	struct manifest *manifest = NULL;
 	int ret = 0;
-	bool retried = false;
+	int retried = 0;
```

Comment:
Can you explain the retry changes in the commit description? Is it a bug fix? etc. It seems to increase the complexity of the code, and it's not obvious to me what benefit it has.